spec/requests/api/v1/messages/index_spec.rb

```diff
@@ -27,5 +27,41 @@
       expect(json[:messages].first[:id]).to eq(messages.first.id)
       expect(json[:messages].first[:content]).to eq(messages.first.content)
     end
+
+    context 'when paginating the messages' do
+      subject(:page_1_subject) do
+        get api_v1_conversation_messages_path(conversation),
+            headers: auth_headers,
+            params: { 'page' => '1' },
+            as: :json
+      end
+
+      subject(:page_2_subject) do
+        get api_v1_conversation_messages_path(conversation),
+            headers: auth_headers,
+            params: { 'page' => '2' },
+            as: :json
+      end
```

Comment:
the same that in the previous pr, we recommend to have only one subject, and adjust the params in each context

Reply:
Thats true, i updated let me know if like this its ok. Thanks!